packages/slice-machine/server/src/api/custom-types/rename.ts

```diff
@@ -14,5 +16,11 @@ export default async function handler(req: RequestWithEnv) {
 
   IO.CustomType.renameCustomType(modelPath, newCustomTypeName);
 
+  IO.Types.upsert(
```

Comment:
So this regenerates all the types ?

Reply:
Correct; it will create the file if it doesn't exist, or overwrite it if it does. The file is regenerated completely each time something is created or updated based on the current Custom Type and Slice models.